docs/pages/appbars/index.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+
+  <DocsPageTemplate>
+
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>
+        App bars are used to provide users a means of navigation through pages in our applications. They allow access to top-level pages, navigate back to a specific page, or dismiss edit forms
+      </p>
+
+      <p>
+        We have four types of app bars: <em>primary bars</em>, <em>back arrow bars</em>, <em>form bars</em>, and <em>bottom bars</em>
+      </p>
+
+      <h3>Primary bars</h3>
+      <p>Primary bars are placed on top level pages in Kolibri. They may house subtabs that lead to different sections of the application</p>
+      <DocsShow>
+        <img class="modal-img" src="./primary-1.png">
+      </DocsShow>
+      <DocsShow>
+        <img class="modal-img" src="./primary-2.png">
+      </DocsShow>
+
+      <h3>Back arrow bars</h3>
+      <p>Back arrow topbars allow users to navigate back to the previous page</p>
+      <DocsShow>
+        <img class="modal-img" src="./back-1.png">
+      </DocsShow>
+
+      <h3>Form bars</h3>
+      <p>Form bars allows users to dismiss a full-screen form and land on the previous page</p>
+      <DocsShow>
+        <img class="modal-img" src="./form-1.png">
+      </DocsShow>
+
+      <h3>Bottom bars</h3>
+      <p>Bottom bars are placed on pages that house editing, content selection, or multi-step processes. They may contain buttons that allow the user to submit specific action(s) or continue with the process. Bottom bars can also contain contextual information and error messaging relevant to the task</p>
+      <DocsShow>
+        <img class="modal-img" src="./bottom-1.png">
+      </DocsShow>
+    </DocsPageSection>
+
+    <DocsPageSection title="Layout" anchor="#layout">
+      <ul>
+        <li>Place navigation icons on the far left</li>
+        <li>Place titles to the right of the navigation icon</li>
+        <li>Place contextual icons to the far right</li>
+      </ul>
+    </DocsPageSection>
+
+    <DocsPageSection title="Titles" anchor="#titles">
+      <p>Appbars must have titles that describe high-level information.</p>
```

Comment:
Do we have guidelines on whether the title refers to the current page vs the page they came from? For example, "create quiz" vs "back to topic"

Reply:
It should always be the current page. I'll adjust the text here